datacite/rest_client.py

```diff
@@ -25,49 +25,35 @@
 class DataCiteRESTClient(object):
     """DataCite REST API client wrapper."""
 
-    def __init__(self, username=None, password=None, url=None, prefix=None,
-                 test_mode=False, api_ver="2", timeout=None):
+    def __init__(self, username, password, url, prefix, timeout=None):
         """Initialize the REST client wrapper.
 
         :param username: DataCite username.
         :param password: DataCite password.
-        :param url: DataCite API base URL. Defaults to
-            https://api.datacite.org/.
-        :param test_mode: Set to True to change base url to
-        https://api.test.datacite.prg. Defaults to False.
+        :param url: DataCite API base URL.
         :param prefix: DOI prefix (or CFG_DATACITE_DOI_PREFIX).
-        :param api_ver: DataCite API version. Currently has no effect.
-            Default to 2.
         :param timeout: Connect and read timeout in seconds. Specify a tuple
             (connect, read) to specify each timeout individually.
         """
         self.username = username
         self.password = password
         self.prefix = prefix
-        self.api_ver = api_ver  # Currently not used
-
-        if test_mode:
```

Comment:
I kind of like test mode doing the url switch. In the old MDS test mode was implemented poorly, but just doing an API URL switch seems like a convenient option to have available.

Reply:
At first, I did not touch it and I liked it too, but then after applying changes as described in the issue https://github.com/inveniosoftware/datacite/issues/46, I could not find the `testMode` param in the DataCite API documentation.
Was it maybe related to the old way of testing with DataCite?